src/CoreWCF.Templates/src/CoreWCF.Templates.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>CoreWCF.Templates</PackageId>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <IncludeContentInPack>true</IncludeContentInPack>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <!-- clear SymbolPackFormat: https://github.com/NuGet/Home/issues/10372 -->
+    <IncludeSymbols>false</IncludeSymbols>
+    <SymbolPackageFormat></SymbolPackageFormat>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" CopyToOutputDirectory="Always" />
+    <Content Remove="templates\Directory.Build.props"/>
+    <Content Remove="templates\Directory.Build.targets"/>
+    <Content Remove="templates\.editorconfig"/>
+    <Compile Remove="templates\**\*"/>
```

Comment:
I'm having difficulty understanding some of these. You are removing the .editorconfig, but I thought we had a separate one so that the generated project included an editorconfig?

Reply:
I see my confusion now. I didn't notice there was separate Content and Compile items. The general convention is to not mix different item types in a single ItermGroup. That always confused me why, but I think I just learnt why. Can you move the Compile into it's own ItemGroup to make things clearer.

Reply:
done. i also added comments